src/storage/v2/disk/storage.cpp

```diff
@@ -1095,56 +940,53 @@ Result<EdgeAccessor> DiskStorage::DiskAccessor::EdgeSetTo(EdgeAccessor * /*edge*
   return Error::NONEXISTENT_OBJECT;
 }
 
-/// TODO: this method should also delete the old key
-bool DiskStorage::DiskAccessor::WriteVertexToVertexColumnFamily(const Vertex &vertex) {
-  MG_ASSERT(commit_timestamp_.has_value(), "Writing vertex to disk but commit timestamp not set.");
-  auto *disk_storage = static_cast<DiskStorage *>(storage_);
+bool DiskStorage::WriteVertexToVertexColumnFamily(Transaction *transaction, const Vertex &vertex) {
```

Comment:
Why are we moving these functions from Accessor to Storage?
*This comment is also for the other functions that have passed from DiskAccessor to DiskStorage.

Reply:
When merging expansions I had to move some functions from accessor level to storage level but didn't move all because I would clutter that PR. That's the first reason: IMO everything should be on the same level.
Second, it's logical to me that this level is Storage because why should Storage::Accessor handle RocksDB related things.